students/17th/team1/donggeunLEE/User/views.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from django.http  import HttpResponse, JsonResponse
+from django.views import View
+
+from .models      import Userinfo
+
+# 회원가입
+class UserSignUpView(View):
+    MINIMUM_PASSWORD_LENGTH = 8
+    
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        password_validity = data['password']
```

Comment:
```suggestion
        password_validity = data['password']
```
이코드도 try안에 안들어가면 KeyError 핸들이 안돼서 익셉션 500 에러가 날겁니다.

Reply:
500 에러 나는거 확인했고, 수정했습니다.!!